internal/localizer/localizer.go

```diff
@@ -0,0 +1,56 @@
+package localizer
+
+import (
+	// Import the internal/translations so that its init() function
+	// is run. It's really important that we do this here so that the
+	// default message catalog is updated to use our translations
+	// *before* we initialize the message.Printer instances below.
+
+	"fmt"
+	"os"
+	"strings"
+
+	_ "github.com/microsoft/go-sqlcmd/internal/translations"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+)
+
+var Translator *message.Printer
+
+var supportedLanguages = map[string]string{
+	"de-de":      "de-DE",
+	"fr-fr":      "fr-FR",
+	"en-us":      "en-US",
+	"zh-hans-cn": "zh-CN",
```

Comment:
>zh-hans-cn [](http://example.com/codeflow?start=2&length=10)

I've never seen "zh-hans-cn" used, just "zh-hans"

Reply:
The bcp47 language tag expects the second half as the region so I was not sure what we should have there. 
Since we are anyway internally mapping it to zh-CN and zh-TW, I have corrected this to be zh-hans and zh-hant for user to specify.